docs/production-setup.md

```diff
@@ -112,5 +114,52 @@ Save pm2 config:
   ```sh
   pm2 save
   ```
+### Using Docker
+
+**Step 1: Update Environment Variables**
+
+Before using Docker, ensure you've configured the environment variables in your `.env` file correctly. Update values such as `MARIADB_DOCKER_PORT`, `REDIS_DOCKER_PORT`, `REDIS_HOST`, and `DB_HOST` as required for your Docker setup.
+
+**Step 2: Start the Docker Containers**
```

Comment:
Before we start we need to build it.

Reply:
It doesn't seem to be necessary for first time as I was able to run everything without the step, and only need to build after making any change. I have added the step anyway.